canvas_sdk/v1/data/command.py

```diff
@@ -25,3 +25,4 @@ class Meta:
     schema_key = models.TextField()
     data = models.JSONField()
     origination_source = models.CharField()
+    anchor_object_dbid = models.BigIntegerField()
```

Comment:
This is the only change I am unsure about. I'm uneasy about this for a few reasons:

1. Users may find anchor models confusing.
2. This field is a Django content types field.
3. It uses a `dbid`, which adds more confusion — users will have to know the subtleties between when to use `id` or `dbid`.

I outlined an alternative here: https://canvasmedical.atlassian.net/browse/KOALA-2293

The alternative idea is to make event contexts as useful as possible, meaning that anchor object IDs (suitable for data model queries) should be included in event contexts where possible.